test/JDBC/input/storedProcedures/Test-sp_set_session_context-vu-verify.mix

```diff
@@ -0,0 +1,217 @@
+-- tsql
+-- Basic Case: Store and Load various types
+EXEC sys.sp_set_session_context 'foo', 'bar'
+go
+
+SELECT sys.session_context(N'foo')
```

Comment:
could you please add three more cases if they are not here?
1. explicitly define a variable of sqlvariant type and test if your function is accepting it
2. use sql variant property function to examine if you are getting same data 
2.1 define a sqlvariant and assign a int 1 
2.3 test returned value from session_context is also of type int 
3. test sessions could not see session context set by other sessions

Reply:
Tests for 1 and 2 added below, 3 should be covered by the different users checking the 'foo' key